Features/SkillTest.feature

```diff
@@ -0,0 +1,76 @@
+﻿Feature: This test suite contains test scenarios for Skill Form Tests
+
+Scenario: A. Create a new skill record		
+	Given I select skills tab
+	Then  I create a new skill records with values  '<skill>' '<level>'
+	Then I should see popup message as '<skill> has been added to your skills'
+	And I should be able to see added skill in table '<skill>' '<level>'
+	Examples: 
+	| skill | level        |
+	| QA    | Intermediate |
+	| C#    | Beginner     |
+
+	Scenario:B. Edit an existing skill to a new skill record	
+	Given I select skills tab
+	Then I Edit an existing skill '<skill>' '<level>'  to a new record with values '<newSkill>' '<newLevel>'
+	Then I should see popup message as '<newSkill> has been updated to your skills'
+	And I should be able to see updated skill in table  '<newSkill>' '<newLevel>'
```

Comment:
same comments apply

Reply:
added new test cases for skill page same as above language